app/assets/javascripts/snowcrash/modules/table.js.coffee

```diff
@@ -0,0 +1,192 @@
+class IndexTable
+  itemName: ''
+  $table: null
+  project: null
+  selectedColumns: []
+  selectedItemsSelector: ''
+  tagColumnIndex: null
+
+  constructor: (@itemName) ->
+    @$table       = $('#index-table table')
+    @$column_menu = $('.dropdown-menu.js-table-columns')
+
+    @selectedItemsSelector = 'input[type=checkbox]:checked.js-multicheck:visible'
+
+    # -------------------------------------------------------- Load table state
+    @loadColumnState()
+    @showHideColumns()
+
+    # -------------------------------------------------- Install event handlers
+    $('#index-table').on('click', '.js-taglink', @onTagSelected)
```

Comment:
tags are something specific to issues
we need to extract that from the class
I would try to inherit the `IndexTable` class and create the `IssuesTable` class with the new functionality

Reply:
✅